src/System.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -158,21 +152,15 @@ protected virtual HashAlgorithmName GetHashAlgorithmName(string algorithm)
 
             switch (algorithm)
             {
-                case SecurityAlgorithms.SHA256:
```

Comment:
This method is always used to "extract"/resolve the digest algorithm from a signature algorithm name (RS256 -> SHA256). I'm not sure it's really useful/semantically correct to include the digest algorithms.